RedditBot.py

```diff
@@ -1,34 +1,22 @@
+import logging
+import os
+
 import discord
 from discord.ext import commands
 from discord_slash import SlashCommand
-from cogs.utilities import Utilities
-import os
-import logging
 from dotenv import load_dotenv
 
+import helpers
+from cogs.utilities import Utilities
 
-# logs data to the discord.log file, if this file doesn't exist at runtime it is created automatically
```

Comment:
where did all the logging setup go?

Reply:
heyah,
I shoved the logging configuration into `helpers.py` to avoid circular imports since functions within that file use logging.

Reply:
nice, yeah just seen that change. Should've reviewed everything first before commenting haha

Reply:
:fingerguns: